### PR TITLE
Updated parser to be in sync with Ben's/Hero Protocol

### PR DESCRIPTION
**Pull request by Zemill**
Updated parser to be in sync with Ben's/Hero Protocol

### CODE REVIEW

File: hots-lambda/aws-lambda-tools-defaults.json [hunk 1]
```diff
@@ -6,16 +6,16 @@
         "dotnet lambda help",
         "All the command line options for the Lambda command can be specified in this file."
     ],
-    "profile"     : "default",
```
Comment (mjibson) [inline hots-lambda/aws-lambda-tools-defaults.json]: This file needs to be reverted.

Comment (Zemill) [reply hots-lambda/aws-lambda-tools-defaults.json]: I didn’t think I made any changes except to the player class and something in the mpq folder.  

Shouldn’t be too hard to replicate

Comment (Zemill) [reply hots-lambda/aws-lambda-tools-defaults.json]: I didn’t think I made any changes except to the player class and something in the mpq folder.  

Shouldn’t be too hard to replicate

Comment (Zemill) [reply hots-lambda/aws-lambda-tools-defaults.json]: Ah I see.  I pushed the pull request after I uploaded it to one of my functions so it changed the defaults.  

My bad.  Just ignore that I suppose

**Pull request closed without merging**